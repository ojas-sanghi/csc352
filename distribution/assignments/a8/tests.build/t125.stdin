23 


28 
5 17 



15 

12 0 
0 


26 19 


4 
19 2 
22 
7 
23 

22 6 
11 
