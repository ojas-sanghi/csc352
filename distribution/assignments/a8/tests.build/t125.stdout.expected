(((-0(-0((-2-)4-)))5(((((-6-)7(-11-))12-)15-)17(-19(-19(-22(-22-))))))23(((-23-)26-)28-))
Insert 10
(((-0(-0((-2-)4-)))5(((((-6-)7((-10-)11-))12-)15-)17(-19(-19(-22(-22-))))))23(((-23-)26-)28-))
